src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -531,6 +542,10 @@ private function registerRouterConfiguration(array $config, ContainerBuilder $co
         $container->setParameter('request_listener.http_port', $config['http_port']);
         $container->setParameter('request_listener.https_port', $config['https_port']);
 
+        if (PHP_VERSION_ID >= 7000 || !method_exists($this, 'addClassesToCompile')) {
+            return;
+        }
```

Comment:
I would wrap the next call instead of returning early here to make the intent clear.